index.js

```diff
@@ -0,0 +1,38 @@
+const menuiconId = document.querySelector('#menu-icon');
+const logoId = document.querySelector('#logo');
+const openmenuId = document.querySelector('#open_menu');
+const portfolioicon = document.querySelector('#portfolio-icon');
+const abouticon = document.querySelector('#about-icon');
+const contacticon = document.querySelector('#contact-icon');
```

Comment:
In conjuction with best practices in naming javascript variables, kindly consider using the camelCase in naming all your variables. For instance, you can name `menuIconId`, `aboutIcon`, `portfolioIcon`, and `openMenuIcon` etc, instead of `menuiconId`, `abouticon`, `portfolioicon`, etc.